packages/manager/apps/hub/src/dashboard/incident/status/routing.js

```diff
@@ -2,7 +2,17 @@ export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('app.dashboard.incident.status', {
     url: '/status',
     component: 'hubIncidentStatus',
+    params: {
+      message: null,
+    },
     resolve: {
+      canResiliate: /* @ngInject */ (ovhFeatureFlipping) =>
+        ovhFeatureFlipping
+          .checkFeatureAvailability('billing:sbgResiliation')
+          .then((featureAvailability) =>
+            featureAvailability.isFeatureAvailable('billing:sbgResiliation'),
+          )
+          .catch(() => false),
```

Comment:
can we create an id variable for `billing:sbgResiliation` to avoid typing two times.
```suggestion
      canResiliate: /* @ngInject */ (ovhFeatureFlipping) => {
        const featureFlippResiliateId = 'billing:sbgResiliation';
        return ovhFeatureFlipping
          .checkFeatureAvailability(featureFlippResiliateId)
          .then((featureAvailability) =>
            featureAvailability.isFeatureAvailable(featureFlippResiliateId),
          )
          .catch(() => false)
          },
```

Reply:
Yes could do but this is meant to be temporary and to be removed at one point so I don't see any issue